File: src/main/java/com/example/ssauc/admin/dto/ReplyDto.java
package com.example.ssauc.admin.dto;

import com.example.ssauc.admin.entity.Admin;
import lombok.*;

@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ReplyDto {
    private long boardId;
    private String title;
    private String content;
    private Admin admin;
}

File: src/main/java/com/example/ssauc/admin/dto/ReplyDto.kt
package com.example.ssauc.admin.dto

import com.example.ssauc.admin.entity.Admin
import lombok.*

@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
class ReplyDto {
    public val boardId: Long = 0
    public val title: String? = null
    public val content: String? = null
    public val admin: Admin? = null
}
